wazuh_vm_tester/src/vm_tester/config.py

```diff
@@ -0,0 +1,394 @@
+"""
+Archivo actualizado de config.py con configuraciones extraídas de los tests
```

Comment:
convert to english

Reply:
Done in [0cbff7b](https://github.com/wazuh/wazuh-virtual-machines/pull/266/commits/0cbff7bf4fc34532c19029684c71e9ef28e4253b)